### PR TITLE
Add QNAP TS-x32PXU/-RP

### PR DESCRIPTION
**Pull request by dsieborger**
Add QNAP TS-{4,8}32PXU-RP, TS-432PXU and their images

### CODE REVIEW

File: device-types/QNAP/TS-432PXU-RP.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+airflow: front-to-rear
+front_image: true
+rear_image: true
+is_full_depth: true
+subdevice_role: parent
+device-bays:
+  - name: Disk 1
+  - name: Disk 2
+  - name: Disk 3
+  - name: Disk 4
```
Comment (harryajc) [inline device-types/QNAP/TS-432PXU-RP.yaml:16]: Disks are not currently being modelled in this repository.

File: device-types/QNAP/TS-432PXU.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+airflow: front-to-rear
+front_image: true
+rear_image: true
+is_full_depth: true
+subdevice_role: parent
+device-bays:
+  - name: Disk 1
+  - name: Disk 2
+  - name: Disk 3
+  - name: Disk 4
```
Comment (harryajc) [inline device-types/QNAP/TS-432PXU.yaml:16]: Disks are not currently being modelled in this repository.

File: device-types/QNAP/TS-832PXU-RP.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+subdevice_role: parent
+device-bays:
+  - name: Disk 1
+  - name: Disk 2
+  - name: Disk 3
+  - name: Disk 4
+  - name: Disk 5
+  - name: Disk 6
+  - name: Disk 7
+  - name: Disk 8
```
Comment (harryajc) [inline device-types/QNAP/TS-832PXU-RP.yaml:20]: Disks are not currently being modelled in this repository.

File: device-types/QNAP/TS-832PXU-RP.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+comments: '[TS-832PXU-RP | Hardware Specs | QNAP](https://www.qnap.com/en/product/ts-832pxu-rp/specs/hardware)'
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 250
+    allocated_draw: 70
+  - name: PSU2
+    type: iec-60320-c14
+    maximum_draw: 250
+    allocated_draw: 70
```
Comment (harryajc) [inline device-types/QNAP/TS-832PXU-RP.yaml:33]: As this has removable PSUs they should be modelled under `module-bays` and then added the required PSU modules if they don't already exists in the `module-bays/QNAP/`

File: device-types/QNAP/TS-432PXU-RP.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+comments: '[TS-432PXU-RP | Hardware Specs | QNAP](https://www.qnap.com/en/product/ts-432pxu-rp/specs/hardware)'
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 250
+    allocated_draw: 54
+  - name: PSU2
+    type: iec-60320-c14
+    maximum_draw: 250
+    allocated_draw: 54
```
Comment (harryajc) [inline device-types/QNAP/TS-432PXU-RP.yaml:29]: As this has removable PSUs they should be modelled under `module-bays` and then added the required PSU modules if they don't already exists in the `module-bays/QNAP/`

File: device-types/QNAP/TS-432PXU-RP.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
+---
+manufacturer: QNAP
+model: TS-432PXU-RP
+slug: qnap-ts-432pxu-rp
+part_number: TS-432PXU-RP
+u_height: 1
+airflow: front-to-rear
```
Comment (harryajc) [inline device-types/QNAP/TS-432PXU-RP.yaml:7]: To have the most complete data possible could you add the `weight` and `weight-unit` values

File: device-types/QNAP/TS-432PXU.yaml [hunk 7]
```diff
@@ -0,0 +1,38 @@
+---
+manufacturer: QNAP
+model: TS-432PXU
+slug: qnap-ts-432pxu
+part_number: TS-432PXU
+u_height: 1
+airflow: front-to-rear
```
Comment (harryajc) [inline device-types/QNAP/TS-432PXU.yaml:7]: To have the most complete data possible could you add the `weight` and `weight-unit` values

File: device-types/QNAP/TS-832PXU-RP.yaml [hunk 8]
```diff
@@ -0,0 +1,46 @@
+---
+manufacturer: QNAP
+model: TS-832PXU-RP
+slug: qnap-ts-832pxu-rp
+part_number: TS-832PXU-RP
+u_height: 2
+airflow: front-to-rear
```
Comment (harryajc) [inline device-types/QNAP/TS-832PXU-RP.yaml:7]: To have the most complete data possible could you add the `weight` and `weight-unit` values

**Review by harryajc — CHANGES_REQUESTED** [review]

File: device-types/QNAP/TS-432PXU.yaml (hunk 2 above)
Comment (dsieborger) [reply device-types/QNAP/TS-432PXU.yaml]: Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/67cdbb458c786b68d02badc850ebf40b2c00a1be

File: device-types/QNAP/TS-832PXU-RP.yaml (hunk 3 above)
Comment (dsieborger) [reply device-types/QNAP/TS-832PXU-RP.yaml]: Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/67cdbb458c786b68d02badc850ebf40b2c00a1be

File: device-types/QNAP/TS-832PXU-RP.yaml (hunk 4 above)
Comment (dsieborger) [reply device-types/QNAP/TS-832PXU-RP.yaml]: Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/67cdbb458c786b68d02badc850ebf40b2c00a1be

File: device-types/QNAP/TS-432PXU-RP.yaml (hunk 5 above)
Comment (dsieborger) [reply device-types/QNAP/TS-432PXU-RP.yaml]: Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/ec98feeb938fde0a23f9abe26b2d8437cb09d275

So far as I can tell, QNAP use off-the-shelf Delta PSUs.  I can't find any sign of a QNAP-specific SKU or P/N.

File: device-types/QNAP/TS-432PXU-RP.yaml (hunk 1 above)
Comment (dsieborger) [reply device-types/QNAP/TS-432PXU-RP.yaml]: Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/67cdbb458c786b68d02badc850ebf40b2c00a1be

File: device-types/QNAP/TS-432PXU-RP.yaml (hunk 6 above)
Comment (dsieborger) [reply device-types/QNAP/TS-432PXU-RP.yaml:7]: Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/2e7c8da0c02c2b630fe4583e36af1b60ee9702b3

File: device-types/QNAP/TS-432PXU.yaml (hunk 7 above)
Comment (dsieborger) [reply device-types/QNAP/TS-432PXU.yaml:7]: Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/2e7c8da0c02c2b630fe4583e36af1b60ee9702b3

File: device-types/QNAP/TS-832PXU-RP.yaml (hunk 8 above)
Comment (dsieborger) [reply device-types/QNAP/TS-832PXU-RP.yaml:7]: Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/2e7c8da0c02c2b630fe4583e36af1b60ee9702b3

**Pull request merged**